#58 - Frontend Layouts

AmonDeShir opened the pull request:
Should close #58.

## Admin
![image](https://github.com/user-attachments/assets/97a81dd5-9540-44c7-b160-659756a02df1)

## Admin - mobile
![image](https://github.com/user-attachments/assets/5aa80f5a-e0ea-4bde-a8b8-9683d668b990)
![image](https://github.com/user-attachments/assets/87082fe9-c4bf-42dc-ad14-fa7516b808c2)

## User
![image](https://github.com/user-attachments/assets/8e3c5db2-70ae-4672-9b1a-8fd63c2bdb19)

## User - mobile
![image](https://github.com/user-attachments/assets/10eb246e-1ae0-4620-9ece-92e11c7408dd)
![image](https://github.com/user-attachments/assets/8334cce7-2bc1-4cfa-b732-2f88e8aeda3c)

kamilpiech97 commented:
conflicts

kamilpiech97 commented:
tests

EwelinaSkrzypacz reviewed (commented):
1. It should be here "Dane o profilu" or something like that instead of "Twoje dane"

![image](https://github.com/user-attachments/assets/b4616090-a3e0-48d0-b4bd-b92a74801064)

2. Should be translated to "Profil"

![image](https://github.com/user-attachments/assets/40921de6-496c-4bad-9286-ecb29c513972)

3. On dashboard I click "Logout" (not Wyloguj) and I see error - maybe just remove this?

![image](https://github.com/user-attachments/assets/dec411c2-ccbd-4b90-8754-1c03a391a65b)

EwelinaSkrzypacz commented:
Conflicts

AmonDeShir commented:
1. fixed 
![image](https://github.com/user-attachments/assets/8b0344c0-89c8-4df1-b5ce-70114a7ba8b6)
2. fixed
![image](https://github.com/user-attachments/assets/a2f819c7-c78b-41b3-bdff-152cd3120b9f)
3. fixed
![image](https://github.com/user-attachments/assets/d93f5280-a65a-481c-acdd-345a49730752)

AmonDeShir commented:
## Admin
![image](https://github.com/user-attachments/assets/e63fb18a-14e1-49a6-b044-ef1e041b856d)
## Super Admin
![image](https://github.com/user-attachments/assets/d27e1fee-221d-42cf-8a1d-ddf351eb995b)

AmonDeShir commented:
Fixed status messages
![image](https://github.com/user-attachments/assets/7ea89b94-3e27-4ac5-bc0b-c92a0d5b619e)

AleksandraKozubal commented on resources/views/app.blade.php:
i suggest to change lang to pl instead of en.

AmonDeShir replied:
done

AleksandraKozubal reviewed (commented):
Logout on email/verify still doesn't work
![image](https://github.com/user-attachments/assets/e291564f-22b1-4c8b-a6f7-0ceec1f0eeaf)

AleksandraKozubal commented on public/favicon.svg:
Do we use it somewhere?

AmonDeShir replied:
No, we don't. Deleted.
